layouts/default/index.md

```diff
@@ -0,0 +1,34 @@
+<!--
+  ~ Copyright 2018 Adobe. All rights reserved.
+  ~ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License. You may obtain a copy
+  ~ of the License at http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software distributed under
+  ~ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  ~ OF ANY KIND, either express or implied. See the License for the specific language
+  ~ governing permissions and limitations under the License.
+  -->
+<!--
+
+note that this is currently neaded, becaue the petridish server doesn't fallback to
+readme.me
+
+-->  
+Helix - {{project.name}}
+=======================
+
+Welcome to Helix. This is a started project to get you going with helix.
+You will find some example templates in the `src` directory. 
+
+Quick Start
+-----------
+
+To compile the scripts and start the development server do:
+
+```bash
```

Comment:
I'd change the content of this to something like "it works!" – this is the thing people will see when `hlx up` completes, so the instructions on building are already covered.